src/js/providers/data-normalizer.ts

```diff
@@ -1,13 +1,18 @@
-import type { GenericObject } from 'types/generic.type';
-
-export type QualityLevel = {
+interface Level {
```

Comment:
Open to suggestions for this.
I was taking a look at how `QualityLevel` and the level argument passed into this function vs. the level argument passed into the `quality-labels` functions was used.
From what I could see, they are slightly different. The levels passed into the `quality-labels` functions seem to come directly from the providers, so the types that Rob mentioned in these comments
https://github.com/jwplayer/jwplayer/pull/3657#discussion_r408290654
https://github.com/jwplayer/jwplayer/pull/3657#discussion_r408314156
I used the `ProviderLevel` interface to represent those.

The levels passed into the `qualityLevel` functions, are provider levels that have been slightly normalized. So they may `PlaylistItemSources` or referred to as jwLevels in the hls.js and shaka files.
I used the `Level` interface to represent those.

I think one of the main differences I noticed was that the `Levels` all have labels, which I believe is important for the `QualityLevels` because I think it would be strange if a `QualityLevel` had an optional label.